src/components/Records/Record/AdditionalInfo/DatasetArray.vue

```diff
@@ -151,6 +137,11 @@ export default {
         }
       }
       return false;
+    },
+    checkRegex(string) {
```

Comment:
Could you please add a comment about what this regex is checking, that would be helpful

Reply:
No problem.